codeship-services.yml

```diff
@@ -27,5 +27,4 @@ app:
         MAILER_USERNAME: test
         MAILER_PASSWORD: test
         NOTIFICATION_EMAIL: test@test.com
-        FROM_NAME: test
```

Comment:
don't need this one anymore?

Reply:
nm, I see you also removed it from local.env.dist so it must have been intentional

Reply:
Good eye. No, it was never used as far as I know. Must have been sitting in my local copy and got added along with these changes.